src/python/pants/core/subsystems/python_bootstrap.py

```diff
@@ -52,7 +53,12 @@ class EnvironmentAware:
 
                 The following special strings are supported:
 
+                For all runtime environment types:
+
                 * `<PATH>`, the contents of the PATH env var
+
+                For only local runtime environments:
```

Comment:
Maybe helpful to be explicit, like ", i.e. when the environment is a `local_environment` target"?